README.md

```diff
@@ -1,30 +1,19 @@
-Confluent's Python Client for Apache Kafka<sup>TM</sup>
```

Comment:
Why this change? I believe the former wording was more legally correct

Reply:
It's an attempt to better match how people will search for this in google. I thought the legal implications were probably negligible - we're not trying to mislead, and the TM is present immediately below.

It may or may not meaningfully help us (exact phrase matches are less important than they once were). I'm pretty sure it won't be worse.

Anyway, I'll change it back.